must-gather/collection-scripts/gather_ceph_resources

```diff
@@ -25,7 +25,6 @@ ceph_commands=()
 ceph_commands+=("ceph auth list")
 ceph_commands+=("ceph balancer dump")
 ceph_commands+=("ceph balancer pool ls")
-ceph_commands+=("ceph balancer status")
```

Comment:
this command exists  in ceph upsteam

```
ceph balancer status
{
    "active": true,
    "last_optimize_duration": "0:00:00.002555",
    "last_optimize_started": "Tue Oct 19 05:54:25 2021",
    "mode": "upmap",
    "optimize_result": "Unable to find further optimization, or pool(s) pg_num is decreasing, or distribution is already perfect",
    "plans": []
}

```

Reply:
I was supposed to remove "ceph balancer dump", fixing it